src/api/index.ts

```diff
@@ -8,6 +8,18 @@ import { BigNumber } from 'bignumber.js'
 
```

Comment:
Chnage import path to `./dutchx` since you renamed it.
Same in `./src/components/BuyButton/index.tsx` and `./src/test/browser/ETH2GNO_Abstract.test.ts`